src/OperationBuilder.cs

```diff
@@ -368,9 +368,11 @@ private Response BuildMethodReturnType(List<Stack<IModelType>> types, IModelType
             {
                 if (response.Schema != null)
                 {
-                    IModelType serviceType = response.Schema.GetBuilder(_swaggerModeler)
+                    
+                    IModelType serviceType = String.IsNullOrEmpty(response.Schema.Reference)?

```

Comment:
Out of curiosity, what does this do?

Reply:
This where I needed your expertise, apparently we simply try to set the model type for a stream as a composite type but I saw there is a test which has the model type as a primitive, simply accounting for that.
Whether that is expected behavior I am not sure

Reply:
Wait, so this is completely independent of your enum change, correct? Or what test are you fixing with that? Why didn't it fail before? Just not sure I'm connecting the dots right now 🙂

Reply:
As in the description of the PR

"Had to make some changes to src/OperationBuilder.cs and src/SwaggerModeler.cs since the latest autorest.common changes breaks the modeler and subsequent tests"

Yes, these changes I had to make to appease the tests but I am not a 100% sure if this is the right thing to do, I can dig up the failing tests if you'd like

Reply:
Ohhhh now I get it - weird, wonder which changes in common would have done that!? 😲

Reply:
Not Quite sure, but I did verify that it was completely independent of modeler/common changes that I introduced